pkgs/build-support/node/fetch-yarn-deps/index.js

```diff
@@ -88,14 +88,15 @@ const isGitUrl = pattern => {
 }
 
 const downloadPkg = (pkg, verbose) => {
-	const fileMarker = '@file:'
-	const split = pkg.key.split(fileMarker)
-	if (split.length == 2) {
-		console.info(`ignoring lockfile entry "${split[0]}" which points at path "${split[1]}"`)
-		return
-	} else if (split.length > 2) {
-		throw new Error(`The lockfile entry key "${pkg.key}" contains "${fileMarker}" more than once. Processing is not implemented.`)
-	}
+	for (let marker of ['@file:', '@link:']) {
+    	const split = pkg.key.split(marker)
+    	if (split.length == 2) {
```

Comment:
Ha not quiet accurate :) Should be 2 tabs here.

Reply:
yikes. should be fine for real now :D